### DISCUSSION TITLE
.whitesource bolt config with renovate enabled only raising security fix PR's

### DISCUSSION THREAD

**Discussion by chris-hinds** [renovatebot/renovate#14441] [Request Help]
I have enabled white source bolt for security scanning/fix Pls. I also would like renovate to be raising standard package bump PR's (according to the white source bolt docs this can be done).

However I am finding that only the remediate security fix PR's are being raised and not the generic package bump PR's. Any idea as to why?

Here is my `.whitesource` config file;

```
{
  "scanSettings": {
    "baseBranches": []
  },
  "checkRunSettings": {
    "vulnerableCheckRunConclusionLevel": "failure",
    "displayMode": "diff"
  },
  "issueSettings": {
    "minSeverityLevel": "LOW",
    "issueType": "DEPENDENCY"
  },
  "remediateSettings": {
    "workflowRules": {
      "enabled": true,
      "minVulnerabilitySeverity": "LOW"
    },
    "enableRenovate": true,
    "extends": [
      "config:base"
    ],
    "enabledManagers": [
      "yarn"
    ],
    "packageRules": [
     ...
    ]
  }
}

```

**Comment by rarkins** [comment]
Could you clarify which app(s) you are running?

Most questions in here are about https://github.com/apps/renovate

Here's WhiteSource Bolt: https://github.com/marketplace/whitesource-bolt. Bolt does not support fix PRs but you mention those

And here's WhiteSource for GitHub.com: https://github.com/apps/whitesource-for-github-com. It's essentially the commercial version of Bolt, which includes fix PRs

**Reply by chris-hinds** [reply]
Ahh good catch, yes I am running `WhiteSource for GitHub.com`

**Comment by rarkins** [comment]
In general, `"enableRenovate": true` is what you have and need to tell the WhiteSource app that you want "update PRs and not just remediation PRs". So that part is OK.

You also have this config:

```
  "enabledManagers": [
      "yarn"
    ],
```

Which is incorrect, because all npm-compatible managers are considered to be "npm" in Renovate config, e.g.

```
  "enabledManagers": [
      "npm"
    ],
```

Please fix that to start with, and then see if there's any change in result.

**Reply by chris-hinds** [reply]
Thanks, will give this change a go.

**Reply by chris-hinds** [reply]
So even after updating the enabledManagers to just `["npm"]` I am still not seeing renovate raise package bump PR's

**Reply by rarkins** [reply]
Are you seeing anything that indicates that it's definitely running at all? e.g. you see some issues or PRs getting updated? I cannot tell which org or repo you're referring to in order to check.

BTW I just noticed that there's something incorrect in your config:

```
 "remediateSettings": {
    "workflowRules": {
      "enabled": true,
      "minVulnerabilitySeverity": "LOW"
    },
```

`workflowRules` are not `remediateSettings`. I think you meant for them to be at the top level of your `.whitesource`?

**Reply by chris-hinds** [reply]
According to the docs [here](https://whitesource.atlassian.net/wiki/spaces/WD/pages/697696422/WhiteSource+for+GitHub.com#Remediate-Settings-(remediateSettings)) `workflowRules` are part of the `remediateSettings`

**Reply by rarkins** [reply]
Sorry, you are right regarding the docs. I was not aware of that setting as it does not impact Remediate/Renovate directly. I suggest that the best next step is to contact WhiteSource Support (who you can feel free to disclose your exact repo name to) and ask them to confirm (a) if the repo is running, and (b) if there are any warnings or errors being logged for it.

**Reply by chris-hinds** [reply]
Ok, Thanks for your help so far @rarkins

**Reply by rarkins** [reply]
Feel free to continue asking questions here once you have any further information. Support is just the best group to contact now that I don't *notice* any config problems.
